wiki/Modifying-Your-Game-with-Scripts.md

```diff
@@ -98,10 +129,58 @@ Hooks.once("init", () => {
 });
 ```
 
-## Changing the Max Level Cap
+## Change the Max Level Cap
 ```js
 /// Increases the Maximum Level for a Player Actor to 30
 Hooks.once("init", () => {
   CONFIG.DND5E.maxLevel = 30;
 });
 ```
+
+## Remove, Rename, and Add new Languages 
+```js
+/// Removes Common, Renames Deepspeech to Voidspeech, and adds a languages called Ochnun
+Hooks.once("init", () => {
+  delete CONFIG.DND5E.languages.common;
+  CONFIG.DND5E.languages.deep = "Voidspeech";
+  CONFIG.DND5E.languages.ochnun = "Ochnun";
+});
+```
+
+## Add new Item Activation Cost Types
+```js
+/// Adds in options to display in the Activation Cost dropdown
+Hooks.once("init", () => {
+  CONFIG.DND5E.abilityActivationTypes.crithit = 'Critical Hit';
+  CONFIG.DND5E.abilityActivationTypes.attack = 'On Attack';
+  CONFIG.DND5E.abilityActivationTypes.attack = 'Replaces Attack';
+  CONFIG.DND5E.abilityActivationTypes.meleehit = 'On Melee Hit';
+  CONFIG.DND5E.abilityActivationTypes.rangedhit = 'On Ranged Hit';
+  CONFIG.DND5E.abilityActivationTypes.weaponhit = 'On Weapon Hit';

```

Comment:
Let's be consistent with use of single- vs. double-quotes.

```suggestion
  CONFIG.DND5E.abilityActivationTypes.crithit = "Critical Hit";
  CONFIG.DND5E.abilityActivationTypes.attack = "On Attack";
  CONFIG.DND5E.abilityActivationTypes.attack = "Replaces Attack";
  CONFIG.DND5E.abilityActivationTypes.meleehit = "On Melee Hit";
  CONFIG.DND5E.abilityActivationTypes.rangedhit = "On Ranged Hit";
  CONFIG.DND5E.abilityActivationTypes.weaponhit = "On Weapon Hit";

```

Reply:
Updated the quotation marks, also noticed "On Attack" and "Replaces Attack" used the same `.abilityActivationTypes.attack` so updated that too